packages/ui/src/components/BarChart/index.tsx

```diff
@@ -29,16 +29,21 @@ type BarChartProps = {
   chartProps?: Partial<BarSvgProps<BarDatum>>
 }
 
+const DEFAULT_MARGIN = { bottom: 50, left: 60, right: 25, top: 50 }
+const DEFAULT_DATA: BarDatum[] = []
+const DEFAULT_AXISFORMATTER = {}
+const DEFAULT_KEYS = ['value']
```

Comment:
Could we use the type from the props?
```suggestion
const DEFAULT_MARGIN: BarChartProps['margin'] = { bottom: 50, left: 60, right: 25, top: 50 }
const DEFAULT_DATA: BarChartProps['data'] = []
const DEFAULT_AXISFORMATTER: : BarChartProps['axisFormatters'] = {}
const DEFAULT_KEYS: : BarChartProps['keys'] = ['value']
```